### DISCUSSION TITLE
Raspberry Pi Pico support ?

### DISCUSSION THREAD

**Discussion by paaaer** [erichelgeson/BlueSCSI-v1#40] [Ideas]
How about Raspberry Pi Pico support instead of blue pill ?
A couple of pros are cheaper and more powerfull.
Should be no problem to compile the code on that platform ?

**Comment by raybellis** [comment]
BlueSCSI uses the 5V tolerant pins on the Blue Pill card.  There are no such pins on the Pico, so level shifters would be required.

Anyone using the Pico would probably also want to use the PIO blocks to implement parts of the SCSI protocol.
